docs/quick-start-guide.md

```diff
@@ -74,7 +74,7 @@ $ kubectl get deployments -n flink-operator
 
 Check the phase and other status attributes in the custom resource
 ```bash
-$ kubectl get flinkapplication.flink.k8s.io -n flink-operator wordcount-operator-example -o yaml
+$ kubectl get flinkapplication.flink.k8s.io -n default wordcount-operator-example -o yaml
```

Comment:
The application is created in `flink-operator` above

Reply:
Ok I missed that. I updated the create command to remove the `-n` flag. I think It's better to let the YAML of a namespaced resource (a `FlinkApplication` in this case) specify its namespace. This removes opportunities of mis-creating it in the wrong namespace and avoids potential RBAC issues.

Reply:
Sorry, I am not sure I follow your comment. I am still wondering why the application is moved to `default` namespace from `flink-operator`. We can just add the namespace `flink-operator` in the yaml file right ?

In the current quick start guide, all the operator resources are in `flink-operator`. To maintain consistency, we added the application also in the same namespace just for quick start. Note that, anyone is free to create applications in any namespace, as long as Operator is monitoring that particular namespace.

Reply:
Oh I just meant that in the sample app YAML, we should have namespace specified. In which namespace doesn't matter. I'll move it back to the `flink-operator` namespace now that you think it's better to be there.